llvm/test/CodeGen/SystemZ/zos-ppa2.ll

```diff
@@ -19,7 +19,7 @@
 ; CHECK:    .quad   L#PPA2-CELQSTRT                 * A(PPA2-CELQSTRT)
 ; CHECK: L#PPA1_void_test_0:
 ; CHECK:    .long   L#PPA2-L#PPA1_void_test_0       * Offset to PPA2
-; CHECK:    .section    "B_IDRL"
+; CHECK:    .section    ".idrl"
```

Comment:
Q: Is the previous "B_IDRL" wrong?

Reply:
My intention was to have unique, uniform synthetic name for the sections. But with Uli's suggestion this is gone now. In the object file/HLASM file, the name is indeed B_IDRL.